terraform/main.tf

```diff
@@ -68,3 +68,16 @@ module "gh_oidc" {
     }
   }
 }
+
+resource "google_service_account" "service_account" {
```

Comment:
I didn't add any private key credentials here, even though they *can* be managed via terraform, because the creds would be stored in plaintext in the terraform state. Seemed nicer to just handle that separately.